pkcs7/Cargo.toml

```
@@ -20,7 +20,9 @@ spki = { version = "0.7" }
 x509-cert = { version = "0.2", default-features = false }
 
 [dev-dependencies]
+der = { version = "0.7", features = ["oid", "pem"], path = "../der" }
 hex-literal = "0.4"
+x509-cert = { version = "0.2", default-features = false, features = ["pem"], path = "../x509-cert" }
```

After #997 this needs to be:

```suggestion
der = { version = "0.7", features = ["oid", "pem"] }
hex-literal = "0.4"
x509-cert = { version = "0.2", default-features = false, features = ["pem"] }
```